.circleci/main.yml

```diff
@@ -2269,3 +2256,11 @@ commands:
             $POETRY_HOME/bin/pip install poetry==1.4.1
             $POETRY_HOME/bin/poetry --version
             ln -s $POETRY_HOME/bin/poetry /home/circleci/bin/poetry
+  install-bazel:
```

Comment:
I'm a little surprised bazel installation doesn't already exist in CI given we are already using it for other stuff. How do those work?

Reply:
We build bazel directly into the Docker image that CI uses.  See etc/ci-image.

Reply:
would that make this step unnecessary then?

Reply:
yeah, if this were a python-only project, the bazel-only image would be sufficient (since bazel grabs python for you whenever you run/test a target that uses python).  after node stuff is in bazel, then these tests can use our CI image.

right now ben is making the choice between installing bazel into the node+python image or installing node on our bazel image.  bazel is easier to install, so this seems like the right choice to me.